Packs/Exabeam/Integrations/Exabeam/Exabeam.py

```diff
@@ -6,18 +6,21 @@
 import dateparser
 import urllib3
 
-
 # disable insecure warnings
 urllib3.disable_warnings()
 
 
 TOKEN_INPUT_IDENTIFIER = '__token'
+DAYS_BACK_FOR_FIRST_QUERY_OF_INCIDENTS = 3
+DATE_FORMAT = '%Y-%m-%dT%H:%M:%S%z'
+TIME_FORMAT_MILISECONDS = '%Y-%m-%dT%H:%M:%S.%f'
```

Comment:
sorry for being petty, but let's call them both `DATETIME_FORMAT` and `DATETIME_FORMAT_MILISECONDS`